app-frontend/src/app/components/sceneDetail/sceneDetail.html

```diff
@@ -0,0 +1,13 @@
+<div class="content">
+  <img ng-if="$ctrl.scene.thumbnails[0]"
+       ng-attr-src="{{$ctrl.scene.thumbnails[0].url}}"
+       class="rounded-img detail-img">
+  <img ng-if="!$ctrl.scene.thumbnails.length"
+       src="http://placehold.it/275x275"
+       class="rounded-img detail-img">
+  <h5 class="color-dark">Image metadata</h5>
+  <dl class="meta-list">
+    <dt ng-repeat-start="(metaKey, metaVal) in $ctrl.scene.sceneMetadata">{{metaKey}}</dt>
```

Comment:
We need to include fields that are outside of the "metadata" but relevant information from the return object. This includes the following:

 - Cloud Cover
 - Acquisition Date
 - Create Time
 - Modification Time
 - Datasource

Maybe some of the others, but at a minimum those ones.